### PR TITLE
CIV-11800 Hide flight delay for unspec

### PR DESCRIPTION
**Pull request by ShwetaTandel-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-11800

### Change description ###
Hide Flight delay option for unspec claims

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: ccd-definition/CaseEventToFields/CreateClaim-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/CaseEventToFields/CreateClaim-nonprod.json:1]: Should we add SDO-R2 in the file name to track it?

File: ccd-definition/CaseField/CaseField-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/CaseField/CaseField-nonprod.json:1]: Should we add SDO-R2 in the file name to track it?
For this one too.

File: ccd-definition/FixedLists/ClaimTypeUnSpec-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/FixedLists/ClaimTypeUnSpec-nonprod.json:1]: Should we add SDO-R2 in the file name to track it?
For this one too.

File: e2e/fixtures/events/createClaim.js
Comment (hmcts-version1-ignacio) [inline e2e/fixtures/events/createClaim.js:1]: Don't we have to add a prod and non-prod file for this?

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

**Pull request merged**